tests/test_snapshot_download.py

```diff
@@ -0,0 +1,19 @@
+import os
+import unittest
+
+from huggingface_hub.snapshot_download import snapshot_download
+
+
+SNAPSHOT_MODEL_ID = "lysandre/pair-classification-roberta-mnli"
```

Comment:
That is a super heavy model. I can setup a lighter version with no issue whatsoever.

Reply:
+1 this would speed up the tests